src/extension/background-script/actions/blocklist/add.js

```diff
@@ -0,0 +1,20 @@
+import db from "../../db";
```

Comment:
We are in the process of switching everything to TS. We should avoid adding ne JS files by now.

Reply:
I don't understand what's needed here. I followed the same approach as many other files in codebase for db. when functionality is finalized, I can adapt it to the latest convention.

Reply:
Rename the file(s) from `.js` to `.ts` and see what happens. If you run into issues, let us know.

Reply:
done. please check.